### PR TITLE
7002 domain names

### PR DESCRIPTION
**Pull request by gdicristofaro**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+    /**
+     * Loads a DomainCategorizer instance using the public suffix list.
+     *
+     * @return The DomainCategorizer instance.
+     * @throws IOException If there is an error reading the file.
+     */
+    private static DomainCategorizer load() throws IOException {
+        InputStream is = DomainCategorizer.class.getResourceAsStream(DOMAIN_LIST);
+        BufferedReader reader = new BufferedReader(new InputStreamReader(is));
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java:92]: Reader should be closed.

File: Core/src/org/sleuthkit/autopsy/coreutils/public_suffix_list.dat [hunk 2]
```diff
@@ -0,0 +1,13494 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/coreutils/public_suffix_list.dat:1]: This file will become public API. Could we move it someplace else?

**Review by dannysmyda — CHANGES_REQUESTED** [review]

File: Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Attempts to get the domain from a url/domain provided removing the
+ * subdomain(s).
+ */
+class DomainCategorizer {
```
Comment (bcarrier) [inline Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java:35]: This isn't really categorization and will be confusing with some of our other uses of categorization. We should call this something like DomainTokenizer.

File: Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java (hunk 1 above)
Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java:92]: Will do.

File: Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java (hunk 3 above)
Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/coreutils/DomainCategorizer.java:35]: Sounds good.

File: Core/src/org/sleuthkit/autopsy/coreutils/NetworkUtils.java [hunk 4]
```diff
@@ -18,21 +18,25 @@
  */
 package org.sleuthkit.autopsy.coreutils;
 
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.UnknownHostException;
-import java.util.StringTokenizer;
+import java.util.logging.Level;
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/coreutils/NetworkUtils.java:25]: Oh, just noticed this. This is not the right logger, could be the coreutils one.

Comment (dannysmyda) [reply Core/src/org/sleuthkit/autopsy/coreutils/NetworkUtils.java:25]: This is the logging level, not logger. My mistake.

**Review by dannysmyda — APPROVED** [review]

**Pull request merged**